snippets/get_transaction.py

```diff
@@ -0,0 +1,49 @@
+"""A snippet that walks us through getting a transaction."""
+from xrpl.asyncio.clients import AsyncWebsocketClient
+from xrpl.models.requests import Ledger, Tx
+
+
+async def async_get_transaction(client: AsyncWebsocketClient) -> None:
+    """
+    Async snippet that walks us through getting a transaction.
+
+    Args:
+        client: The async network client to use to send the request.
+
+    Raises:
+        Exception: if meta not included in the transaction response.
+    """
+    await client.open()
```

Comment:
If you do want to use the websocket clients, you should use `with ... as ...` syntax - the websocket client documentation explains details.